telco-hub/configuration/reference-crs-kube-compare/optional/quay/quayNS.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Omit all quay until published.

Reply:
There is a pending PR to remove quay from the repo until we've verified. We should remove it here too.